fava/static/package.json

```diff
@@ -30,6 +30,7 @@
     "jquery-dragster": "github:catmanjan/jquery-dragster",
     "jquery-query-object": "^2.2.3",
     "jquery-stupid-table": "github:medsabir/Stupid-Table-Plugin#feat-npm-publish",
-    "mousetrap": "^1.5.3"
+    "mousetrap": "^1.5.3",
+    "query-string": "^4.1.0"
```

Comment:
This adds a second JS dependency to deal with the query string. Do we really need both?